repository/Grease-Squeak6-Core.package/GRSqueakPlatform.class/instance/writeFileStreamOn.do.binary..st

```diff
@@ -0,0 +1,8 @@
+as yet unclassified
+writeFileStreamOn: aString do: aBlock binary: aBoolean 
+	| stream |
+	stream := aBoolean
+				ifTrue: [ (MultiByteFileStream fileNamed: aString) ascii; wantsLineEndConversion: true; yourself ]
```

Comment:
Isn't this the wrong way around? When `aBoolean` is `true`, a binary stream is being requested.

Reply:
You are right. Thanks for spotting!
I just copy/pasted the method from the Squeak5 package without actually checking it...